app/forms/steps/conviction/conviction_subtype_form.rb

```diff
@@ -19,12 +19,18 @@ def changed?
         disclosure_check.conviction_subtype != conviction_subtype
       end
 
+      # Motoring penalty points always have an endorsement
+      def motoring_endorsement
+        GenericYesNo::YES if ConvictionType.new(conviction_subtype).motoring_penalty_points?
+      end
+
       def persist!
         raise DisclosureCheckNotFound unless disclosure_check
         return true unless changed?
 
         disclosure_check.update(
           conviction_subtype: conviction_subtype,
+          motoring_endorsement: motoring_endorsement,
```

Comment:
It seems to me there is nothing in the service that requires this value to be set so, following the philosophy "no code, no bugs" I would recommend to remove this completely 😃 

The value will be set in the journey that asks the endorsement question, and any other journey will be `nil`.

Only difference is, in the results page, the line "Motoring endorsement: yes" will not show, but I think that's actually correct, as usually it is more confusing for the users to see information that they've not been asked for, specially if in this case is superfluous.

I would also add the `motoring_endorsement: nil` at the end of the list of dependent attributes to be reset, because if a user enters a motoring journey that asks this question but then go back and change to any other journey, the `motoring_endorsement` attribute might have been set and affect calculations or be confusing in the results page.

There might actually be more motoring attributes to be reset now that I see the list, but that come be done in a separate PR.

Reply:
Yep, happy with "no code no bugs" philosophy!
I've made these changes.

I've also taken note on reseting the other motoring attributes and can address them in another PR.